app/js/lib/foxbox/network.js

```diff
@@ -183,19 +186,26 @@ export default class Network extends EventDispatcher {
    * went offline following by another event indicating that box is online
    * again.
    *
+   * @return {Promise}
    * @private
    */
   [p.pingBox]() {
     const previousState = this[p.local] || this[p.remote];
 
-    this[p.ping](`${this.localOrigin}/ping`)
-      .then((isOnline) => this[p.onPong](previousState, p.local, isOnline));
+    const pingPromises = [
+      this[p.ping](`${this.localOrigin}/ping`)
+        .then((isOnline) => this[p.onPong](previousState, p.local, isOnline)),
+    ];
 
     // @todo Find a better way to detect if a tunnel connection is active.
     if (this[p.settings].tunnelOrigin) {
-      this[p.ping](`${this.tunnelOrigin}/ping`)
-        .then((isOnline) => this[p.onPong](previousState, p.remote, isOnline));
+      pingPromises.push(
+        this[p.ping](`${this.tunnelOrigin}/ping`)
+          .then((isOnline) => this[p.onPong](previousState, p.remote, isOnline))
+      );
     }
+
+    return Promise.all(pingPromises);
```

Comment:
> Yes please can you try to do that in that PR?

Okay, trying.;

> Also different periods of time sound good.
Maybe also we should use the connection type from the net-info API (only in Chrome for now). If on Wifi we can ping more often that on radio. This optimisation can definitely wait for another PR as long as we fill a bug for it.

Done https://github.com/fxbox/app/issues/135